pom.xml

```diff
@@ -169,6 +170,12 @@
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Not sure this dependency will be needed once the BadRequestException is replaced with our own